cluster/cluster_test.go

```diff
@@ -49,20 +50,29 @@ func TestEncode(t *testing.T) {
 					{
 						Address:         testutil.RandomETHAddress(),
 						ENR:             fmt.Sprintf("enr://%x", testutil.RandomBytes32()),
-						ConfigSignature: testutil.RandomSecp256k1Signature(),
-						ENRSignature:    testutil.RandomSecp256k1Signature(),
+						ConfigSignature: []byte{},
```

Comment:
Suggest adding them as that is default, then removing if version doesn't support it.